api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -211,3 +212,98 @@ def create_offers_for_each_subcategory() -> None:
                 offer__subcategoryId=subcategory.id,
                 quantity=random.randint(10, 100),
             )
+
+
+def create_offers_with_same_author() -> None:
+    # an author with 12 different books
+    author = Fake.name()
+    venues = [
+        offerers_factories.VenueFactory(
+            name="same author " + str(venue["name"]),
+            venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+            isPermanent=True,
+            latitude=venue["latitude"],
+            longitude=venue["longitude"],
+            address=venue["address"],
+            postalCode=venue["postalCode"],
+            city=venue["city"],
+            departementCode=venue["departementCode"],
+        )
+        for venue in random.choices(venues_mock.venues, k=4)
+    ]
+    for venue in venues:
+        for _ in range(4):
+            create_offer_with_ean(Fake.ean13(), venue, author=author)
+
+    # an author with a single book in  a single venue
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Une fonction par commentaire rendrait le code un peu plus lisible selon moi 

```python
def create_offers_with_same_author():
    create_author_with_multiple_books(book_count=12)
    create_author_with_one_book_in_multiple_venues(venue_count=4)
    # ...
```

Avec le même code que tu as ici
Medium logaf, on peut discuter :wink: